test/minitest_ext.rb

```diff
@@ -16,7 +16,10 @@ def setup
     end
 
     def run_cmd(cmd)
-      ShopifyCli::EntryPoint.call(cmd.split(' '), @context)
+      # fixing flakey tests caused by ShopifyCli::Core::Monorail.log.invocation calling Git.sha/Git.exec
+      ShopifyCli::Git.stubs(:sha).returns("bb6f42193239a248f054e5019e469bc75f3adf1b")
```

Comment:
I added this temp fix to address some of the flakey tests I've been seeing:
![image](https://user-images.githubusercontent.com/30087610/78606302-146f8080-782b-11ea-888a-c4dcd305ba5f.png)
We can alternately address this by not having the tests execute `EntryPoint` but that's a much bigger overhaul.